app/controllers/videos_controller.rb

```diff
@@ -1,8 +1,20 @@
 # frozen_string_literal: true
 
 class VideosController < ApplicationController
+  def index
+    authorize Video
+    return if search_params[:search].blank?
```

Comment:
Rubocop didn't let me use `present?` in this case

Reply:
Yeah, it's better like they recommend ;)